GitHub.md

```diff
@@ -15,6 +15,8 @@ We follow the "typical" GitHub workflow when contributing changes:
    title of your pull request is descriptive. If you are fixing an issue that exists in our bug tracker reference it
    like this: `MBS-4635: Allow replacing images`. **Not** `[MBS-4635] - Allow replacing images` or `Allow replacing images
    (MBS-4635)` or `MBS-4635`.
+   If you are fixing several issues at once, reference it like this: `MBS-9477, MBS-10568, MBS-10569: DiscID tab improvements`.
```

Comment:
Does it need to be comma-separated? (I've generally done slash-separated, but I can do comma if the automated bot needs it)

Reply:
I just picked the most common syntax I found among MeB repositories, since that is simpler to give a single example. Actually, any separator in `&+,/;|` is currently matched. The “bot” account is just executing an automation rule I wrote from scratch, it can match anything we want to.